### ISSUE TITLE
Loss

### ISSUE THREAD

**Issue by hongzimao** [ikostrikov/TensorFlow-VAE-GAN-DRAW#16]
Shouldn't it be reduced_mean instead of reduced_sum when computing the vae and reconstruction loss?

**Comment by atabakd** [comment]
Yes, but if you use reduced_sum for **all** losses consistently, then it shouldn't matter (mean is basically sum, divided by a constant)
